Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -2049,12 +2049,8 @@ private void CheckIfTxPolicyExceptionHasInnerException()
                 {
                     Assert.NotNull(e.InnerException);
 
-                    // This is visual representation of the test.
-                    // In actual use-case, this debugging lines do
-                    // not show.
```

Comment:
Should these lines be removed?  What did you intend?

Reply:
My thought was:
- The contents of the comments are ambiguous; e.g. why should the lines not appear?
- Also, the following `Debug` lines are quite self-explanatory: they print the exception.

Therefore, the comments deemed as unnecessary.